### ISSUE TITLE
Training with VCTK corpus

### ISSUE THREAD

**Issue by VladimirMaksovic** [NVIDIA/tacotron2#187]
Hi,

Before going into details, I would like to point out two things:
 - I'm really new in this ML stuff, so please bear with my ignorance
 - I'm aware of two similar issues already closed (#167 & #121) but I could not figure out details I need over there

I used tacotron2_statedict.pt and --warm_start to train with VCTK corpus. Idea was to change the speaker with limited amount of data. As mentioned in #121 VCTK contains 400 sentences spoken by 109 speakers. I trained with one speaker at the time with following command:
`python train.py --output_directory=outdir --log_directory=logdir -c tacotron2_statedict.pt --warm_start --hparams="training_files=filelists/vctk_p226_text_train_filelist.txt,validation_files=filelists/vctk_p226_text_val_filelist.txt"`

I prepared the training data in two steps:
 - Split the data in 80% training and 20% validation
 - Convert the data so that the silence is removed (as you recommended in #121) with sox (sox  in.wav out.wav rate 22050 silence 1 '0.1 '1% reverse silence 1 0.1 1% reverse)

I changed following parameters:
 - epochs=5000  - to see how the training behaves on the long run
 - p_attention_dropout=0.001, p_decoder_dropout=0.001  - blind man trying to avoid overfitting
 - learning_rate=1e-4 - according to your comment on #167 (train longer and anneal the learning rate to reduce the validation loss)
 - batch_size=46 - I have Quadro P5000 with 16GB of RAM so I had to reduce the batch size

Results for the p226 speaker are attached: [vctk_inference.zip](https://github.com/NVIDIA/tacotron2/files/3093691/vctk_inference.zip)

As far as I could say, this seems okayish (speaker voice and pronunciation is kinda close), but there is a lot of this robotic "noise".

Do you have some idea how to improve the results?

Tensorboard pages: 
![scalars](https://user-images.githubusercontent.com/18186406/56352257-1c27b680-61cf-11e9-961a-f34fab339908.png)
![images](https://user-images.githubusercontent.com/18186406/56352264-20ec6a80-61cf-11e9-8069-2a87a97735b9.png)

**Comment by rafaelvalle** [comment]
These samples you shared were produced with the model at which iteration?

**Comment by VladimirMaksovic** [comment]
The samples are named as checkpoint_<**iteration**>.wav
I did not include 2k iteration sample, but there is 3k one. It is a bit weird, but subjectively I think that from 4k iterations it sounds best and then it stays the same, or gets a bit worse.

I used WaveGlow if that is what you are referring to.
I have now figured out that you have changed WaveGlow model, so I will also try with that one.

Does the hyper params changes I did makes sense to you? What would be your suggestion to improve the results?

**Comment by rafaelvalle** [comment]
The best validation loss is at 2k iterations and it should give you the model that best generalizes to data outside the training data. 

You might be able to improve your validation loss by increasing dropout, not decreasing it as you did.

You can probably benefit from fine-tuning our shared waveglow to the speaker you trained on.

**Comment by VladimirMaksovic** [comment]
Thanks for your fast response. I will play around with your suggestions. 

For now I would close this topic.

**Comment by rafaelvalle** [comment]
Great, let us know how it goes!

**Comment by anderleich** [comment]
How can we fine-tune the shared waveglow? Is there something as warm_start in tacotron2?
